benchmarks/unconstrained/run_benchmarks.jl

```diff
@@ -0,0 +1,45 @@
+using Optim, OptimTests, CUTEst, ProgressMeter, Plots, DataFrames, BenchmarkTools
+
+do_benchmarks = false
+saveplots = false
+
+pkg_dir = Pkg.dir("OptimTests")
+version_sha = latest_commit("OptimTests")
+benchmark_dir = pkg_dir*"/benchmarks/unconstrained/history/"
+version_dir = benchmark_dir*version_sha
+try
+    run(`mkdir $version_dir`)
+catch
+
+end
+cd(version_dir)
+
+default_names = ["Accelerated Gradient Descent",
+                 "BFGS",
+                 "L-BFGS",
+                 "Momentum Gradient Descent",
```

Comment:
This position in default_solvers points to `ConjugateGradient()`, not to `MomentumGradientDescent()`